example-nextjs/components/CodeRunnerUI.tsx

```diff
@@ -0,0 +1,124 @@
+            strategy="beforeInteractive"
+          />
+          <main className="mx-auto mb-12 max-w-7xl sm:mt-12">
+            <div className="text-left">
+              <h1 className="text-3xl tracking-tight text-gray-900 dark:text-white sm:text-5xl md:text-5xl">
+                <span className="block font-mono xl:inline">
+                  Embed executable code snippets on your site
+                </span>
+              </h1>
+              <p className="max-w-md mt-4 text-base text-gray-500 dark:text-gray-450 md:mx-auto sm:text-lg md:mt-16 md:text-xl md:max-w-3xl">
+                <b>run-wasm</b> is an api which allows you to easily execute
+                code via WebAssembly based programming languages. <br />
+                <br /> It allows you to include interactive code examples in
+                your website.
+              </p>
+              <GithubButton />
+            </div>
+          </main>
+
+          {/* <RunWasm language="Python" code={inputCode} /> */}
+
+          <div>
+            <label className="block pb-4 text-sm font-medium text-gray-700 dark:text-gray-450">
+              {props.languageLabel}
+            </label>
+
+            <div className="mt-1 ">
+              <div className="relative group">
+                <div className="absolute -inset-0.5 dark:bg-gradient-to-r from-indigo-300 to-purple-400 rounded-lg blur opacity-25 group-hover:opacity-100 transition duration-1000 group-hover:duration-200 animate-tilt" />
+                <Editor
+                  height="20rem"
+                  defaultLanguage="python"
+                  defaultValue={inputCodeRef.current}
+                  onChange={(value) => {
+                    inputCodeRef.current = value
+                  }}
+                  className="block w-1/2  text-white bg-gray-900 border-gray-300 rounded-lg   shadow-sm p-0.5 border   dark:border-purple-300 focus:ring-gray-500 focus:border-gray-500 sm:text-sm"
+                  theme="vs-dark"
+                  options={{ fontSize: 12 }}
+                />
+              </div>
+            </div>
+          </div>
+
+          <div className="pt-8 ">
+            <div className="grid items-start justify-left">
+              <div className="relative group">
+                <button
+                  className="relative flex items-center py-4 leading-none bg-black divide-x divide-gray-600 rounded-lg px-7 border-gray-300 disabled:bg-gray-700 disabled:cursor-not-allowed"
+                  onClick={() => runCode(inputCodeRef.current)}
+                  disabled={props.hideRunCode}
+                >
+                  <span className="text-gray-100 transition duration-200 group-hover:text-gray-100">
+                    {!props.hideRunCode
+                      ? 'Run Code →'
+                      : `Loading ${props.languageLabel}...`}
+                  </span>
+                </button>
+              </div>
+            </div>
+          </div>
+
+          {props.children}
```

Comment:
question: isn't `props.children` optional? Do we need to do this? Or is that already handled by React for us?
```suggestion
          {props.children || null}
```

Reply:
This is already handled by React. Any falsey value (except 0 is not rendered).